lib/main_common.pm

```diff
@@ -1098,13 +1098,11 @@ sub load_consoletests {
         loadtest "console/supportutils";
     }
     loadtest "console/check_package_version" if check_var('ORIGINAL_TARGET_VERSION', '12-SP5');
-    loadtest "console/force_scheduled_tasks" unless is_jeos;
     if (get_var("LOCK_PACKAGE")) {
         loadtest "console/check_locked_package";
     }
     loadtest "console/textinfo";
     loadtest "console/rmt" if is_rmt;
-    loadtest "console/hostname" unless is_bridged_networking;
```

Comment:
This looks unrelated

Reply:
It is related as `hostname` is already scheduled from `load_system_prepare_tests()` function and as I added the `load_system_prepare_tests();` function to `load_reboot_tests` the `hostname` module would be scheduled twice.